novem/utils.py

```diff
@@ -337,3 +343,35 @@ def _data_ready() -> Result:
 
     ctnt = "".join(sys.stdin.readlines()) if has_data else ""
     return ctnt if ctnt else None
+
+
+def migrate_config_04_to_05(path: str, config: configparser.ConfigParser, co: Config) -> bool:
+    # before: there is no `version` key in the [app:cli] section, or the version is less than 0.5
+    # action: insert version, and rewrite all `api.novem.no` URLs to `api.novem.io'
+
+    try:
+        version = Version(config["app:cli"]["version"])
+    except (KeyError, InvalidVersion):
+        version = Version("0.4")
+
+    if version >= Version("0.5"):
+        # we're good!
+        return False
+
+    # set this to 0.5 instead of the actual version to allow for stacked migrations in the future
+    if not config.has_section("app:cli"):
+        config.add_section("app:cli")
+
+    config["app:cli"]["version"] = "0.5"
+
+    for section in config.sections():
```

Comment:
This includes all the user `[profile:<profile_name>]` sections as well?

Reply:
Yup.